lib/rails_admin_nestable/helper.rb

```diff
@@ -19,7 +19,7 @@ def nested_tree_nodes(tree_nodes = [])
     end
 
     def action_links(model)
-      content_tag :ul, class: 'inline actions' do
+      content_tag :ul, class: 'inline list-inline' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.